index.js

```diff
@@ -3,15 +3,53 @@ const path = require('path')
 const PORT = process.env.PORT || 5000
 const eventsController = require('./controllers/events');
 
+<<<<<<< HEAD
+//For local testing
+//Guarded include for dotenv as it is not a production dependency
+if(process.env.NODE_ENV!="production"){
+  console.log(`Running locally in ${process.env.NODE_ENV}`);
+  const env = require('dotenv');
+  env.config();
+  if(env.error) throw env.error;
+}
+
+const { Pool } = require('pg');
+var pool;
+const constring = process.env.DATABASE_URL || `postgres://${process.env.DB_USER}:${process.env.DB_PASS}@localhost/grababite`;
+
+pool = new Pool({
+  connectionString: constring
+});
+
+=======
+>>>>>>> 7a749faad9ed8071998e154551088c0b3ac702d6
 const app = express();
 
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
+
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
 
 app.get('/', (req, res) => res.render('pages/index'))
 
+app.get('/restaurant/:uid', (req, res) => {
+  var uid = req.params.uid;
+  console.log(uid);
+  var query = `select * from restaurants where id=${uid}`;
+
+  pool.query(query, (error, result)=>{
+    if(error)
+      res.end(error);
+      
+    console.log(result);
+    var results = {'attributes':result.rows[0]};
```

Comment:
1. With the new changes to the partials, a pageTitle: 'appropriate title' and path '/restaurant/id' should be appended to the 'results' object passed to the render function, this is a breaking change
2. Error handling is not handled eloquently getting an ejs compiler issue log where attributes.name is undefined, this is not a breaking change